README.md

```diff
@@ -202,6 +202,65 @@ You can use this to remove scrolling on the body while the modal is open.
 }
 ```
 
+### Refs
+
+You can use ref callbacks to get overlay and content DOM nodes
+
+```jsx
```

Comment:
I think it would be better to have a small `README.md`. Better move to a section on the `docs`.